pkg/clicmd/root.go

```diff
@@ -106,8 +106,8 @@ func NewRoot(appFs afero.Fs, wd string, args []string) (*cobra.Command, error) {
 		return nil, err
 	}
 
-	if len(args) > 0 && cmdName != "init" {
-		a, err = app.Load(appFs, wd, false)
+	if cmdName == "version" {
```

Comment:
Where will non-`version` commands have their app loaded if not here?

Reply:
Disregard; something weird happened with having VS code and vim opening the same file

Reply:
This has bit me in the past - if you're using a neovim variant, you may need to add the following to your ~/.vimrc to get it to auto-reload external changes when it receives focus:

```
" autoload external changes
set autoread
au FocusGained * :checktime
```